web/src/compose_validate.js

```diff
@@ -244,7 +246,10 @@ function show_wildcard_warnings(stream_id) {
 
     // only show one error for any number of @all or @everyone mentions
     if ($(`#compose_banners .${CSS.escape(classname)}`).length === 0) {
-        compose_banner.append_compose_banner_to_banner_list(wildcard_template);
+        compose_banner.append_compose_banner_to_banner_list(
+            wildcard_template,
+            $("#compose_banners"),
+        );
```

Comment:
I guess for this one, I think it's a bug we recently opened an issue for that we don't call it when editing a message, but I think I agree to do this here.

Reply:
This is https://github.com/zulip/zulip/issues/25411 and https://github.com/zulip/zulip/issues/25412.